src/WMBot/Commands/Channel.cs

```diff
@@ -167,6 +167,7 @@ public static void PartChannel(Channel channel, string user, string host, string
                         lock (Configuration.Channels)
                         {
                             channel.Remove();
+                            IRC.DeliverMessage("Successfully parted " + channel_name + ".");
```

Comment:
Does this compile? From what I know DeliverMessage requires at least one argument, which is a target (channel / user). Also DeliverMessage should be called before Remove as delivery of messages to channels in which bot isn't present is not supported (it's already unknown channel after that call).

Moreover, what is a purpose of this? Where is the message even going to be sent to? To original channel the bot parted? Why? The information is already displayed there. Or is that supposed to be echoed to place from which the command was used?

Thanks